internal/scanner/scanner.go

```diff
@@ -251,15 +251,15 @@ func (s *Scanner) auditResource(ctx context.Context, policies []*policies.Policy
 			errored = true
 		} else {
 			log.Debug().Dict("response", zerolog.Dict().
-				Str("uid", string(auditResponse.Response.UID)).
+				Str("uid", string(admissionReviewResponse.Response.UID)).
 				Str("policy", policy.GetName()).
 				Str("resource", resource.GetName()).
-				Bool("allowed", auditResponse.Response.Allowed),
+				Bool("allowed", admissionReviewResponse.Response.Allowed),
 			).
 				Msg("audit review response")
 		}
 
-		report.AddResultToPolicyReport(policyreport, policy, auditResponse.Response, errored)
+		report.AddResultToPolicyReport(policyreport, policy, admissionReviewResponse, errored)
```

Comment:
@flavio As an alternative, we could check if `admissionReviewResponse` is nil in `report.AddResultToPolicyReport` instead of using the `errored` boolean. wdyt?

Reply:
You mean, to drop the bool from the `AddResultToPolicyReport` and just check for the `admissionReviewResponse being != nil`? I would prefer to have the bool flag around. I also think it's fine to have a dedicated variable (`errored`) around, it makes everything more explicit